### DISCUSSION TITLE
Eingabe von Einschreibeschlüsseln verbessern

### DISCUSSION THREAD

**Discussion by wie0815** [lotta-schule/web#260] [Ideen]
### Bereich

Nutzerprofil

### Verhalten

In der Regel geben Nutzer die Schlüssel ein und klicken auf Speichern.
Somit werden die Schlüssel nicht übernommen, dafür muss man vorher auf ENTER drücken

Aussehen / Design

### Problem

Bei der Eingabe der Einschreibeschlüssel gibt es immer wieder Probleme. Statt dem aktuellen Text im Dialog (vgl. Bildschirmfoto)
<img width="920" alt="Bildschirmfoto 2024-06-10 um 19 46 44" src="https://github.com/lotta-schule/web/assets/143288974/ac40eda0-5319-438a-8a6f-cdeb6095b245">
... schlage ich folgenden Text vor:
Überschrift: **Meine Einschreibeschlüssel** (bleibt so)
unter der Überschrift: **Gib Deinen Einschreibeschlüssel ein und bestätige die Eingabe mit ENTER!** (Neu)
In das Feld: **Mehrere Schlüssel mit Komma trennen und mit ENTER bestätigen. Anschließend SPEICHERN!**

### Beschreibung

In der Regel geben Nutzer die Schlüssel ein und klicken auf Speichern. Somit werden die Schlüssel nicht übernommen!

**Comment by techotline** [comment]
Ich schließe mich ohne Widerspruch seiner Idee an. Oftmals wird nachgefragt, ob und wie ein Einschreibeschlüssel funktioniert. Das ist zwar nicht ganz die 'Einschränkung' von uns, weil normalerweise das System anwenderfreundlich aufgebaut und grundsätzlich einfach zu verstehen ist, jedoch fehlt hier eine kleine Beschreibung, um alle User abzuholen, die dem Konzept hinter einer 'Rolle' nicht folgen können. Nicht alle können das Know-how haben, wie wir es besitzen.

**Comment by ptitmouton** [comment]
Wir speichern auch Sachen, die noch im Textfeld stehen

**Comment by ptitmouton** [comment]
Hab es als Ticket #292 angelegt.

Ich denke das Thema ist vorerst ausdiskuttiert
